### PR TITLE
docs: fix typo in README

### PR DESCRIPTION
**Pull request by Manice18**
Correct multiple spelling checks in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Go to [http://localhost:3000](http://localhost:3000) in your browser.
 Volto is actively developed since 2017 and used in production since 2018 on the following websites:
 
 - [VHS Ehrenamtsportal](https://vhs-ehrenamtsportal.de) (Website to help volunteers that help refugees for the [German Adult Education Association](https://www.dvv-vhs.de/en/home/), developed by [kitconcept GmbH](https://kitconcept.com), 2018)
-- [Zeelandia](https://zeelandia.de) (Corporate website for one of the leading backery ingrediences manufactors in Germany, developed by [kitconcept GmbH](https://kitconcept.com), 2019)
+- [Zeelandia](https://zeelandia.de) (Corporate website for one of the leading backery ingredients manufacturers in Germany, developed by [kitconcept GmbH](https://kitconcept.com), 2019)
```
Comment (davisagli) [inline README.md:141]:
```suggestion
- [Zeelandia](https://zeelandia.de) (Corporate website for one of the leading bakery ingredients manufacturers in Germany, developed by [kitconcept GmbH](https://kitconcept.com), 2019)
```

**Review by davisagli — APPROVED** [review]
Thanks for the fixes! I noticed one more that is needed.

**Comment by davisagli** [comment]
Note: Normally it is necessary to include a changelog entry (in the news folder) and to have a signed Plone contributor agreement, in order for a PR to be merged. However, I bypassed those requirements since the fixes are trivial.

**Comment by Manice18** [comment]
@davisagli i was finding the agreement doc but wasn't able to
thanks for the merge
just had a query which is whether our org will be a part of gsoc this year or not ?

**Comment by stevepiercy** [comment]
@Manice18 see:

- https://plone.org/foundation/contributors-agreement
- https://plone.org/community/gsoc

**Comment by Manice18** [comment]
thanks for the info @stevepiercy 
i am very much eager to contribute into your organization 
hope I'll make it

**Pull request merged**